examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_file_path_input.py

```diff
@@ -0,0 +1,12 @@
+import importlib
+from pathlib import Path
+from promptflow import tool
+from promptflow.contracts.types import FilePath
+
+
+@tool
+def my_tool(input_file: FilePath, input_text: str) -> str:
+    # customise your own code to handle and use the input_file here
+    new_module = importlib.import_module(Path(input_file).stem)
+
+    return new_module.hello(input_text)
```

Comment:
In flow.dag you use `convert_image_to_data_url.py` as example, but this py does not have `hello` function?